package.json

```diff
@@ -91,11 +91,12 @@
     "mux.js": "4.3.2",
     "url-toolkit": "^2.1.3",
     "video.js": "^6.2.0",
-    "webworkify": "1.0.2"
+    "webworkify": "^1.5.0"
   },
   "devDependencies": {
-    "babel": "^5.8.0",
-    "babelify": "^6.0.0",
+    "babel-cli": "^6.26.0",
+    "babel-preset-env": "^1.6.1",
```

Comment:
does the default presets match our supported browser list?

Reply:
I think instead we should explicitly use the presets and plugins we need, see the .rc below